frontend/src/components/Plan/hook/fetchAllPlanData.ts

```diff
@@ -0,0 +1,56 @@
+import { useEffect, useState } from 'react'
```

Comment:
This filed was renamed from fetchPlanData. It will fetch contacts data. It can only run one time.